app/controllers/katello/api/v2/content_imports_controller.rb

```diff
@@ -0,0 +1,54 @@
+module Katello
+  class Api::V2::ContentImportsController < Api::V2::ApiController
+    before_action :find_content_view, :only => [:version]
+    before_action :find_organization, :only => [:library]
+    before_action :find_default_content_view_version, :only => [:library]
+
+    api :POST, "/content_imports/version", N_("Import a content view version")
+    param :content_view_id, :number, :desc => N_("Content view identifier"), :required => true
+    param :path, String, :desc => N_("Directory containing the exported Content View Version"), :required => true
+    param :metadata, Hash, :desc => N_("Metadata taken from the upstream export history for this Content View Version"), :required => true
+    def version
+      if @view.default?
+        fail HttpErrors::BadRequest, _("Cannot use this end point for importing to library. "\
+                                       "If you intented to upload to library, use the library endpoint.")
+      end
+
+      task = async_task(::Actions::Katello::ContentViewVersion::Import, @view, path: params[:path], metadata: metadata_params&.to_h)
+      respond_for_async :resource => task
+    end
+
+    api :POST, "/content_imports/library", N_("Import a content view version to the library")
+    param :organization_id, :number, :desc => N_("Organization identifier"), :required => true
+    param :path, String, :desc => N_("Directory containing the exported Content View Version"), :required => true
+    param :metadata, Hash, :desc => N_("Metadata taken from the upstream export history for this Content View Version"), :required => true
+    def library
+      task = async_task(::Actions::Katello::ContentViewVersion::ImportLibrary, @organization, path: params[:path], metadata: metadata_params&.to_h)
+      respond_for_async :resource => task
+    end
+
+    private
+
+    def find_content_view
```

Comment:
As a reader who's looking at the before_action block above, naming this method `find_publishable_content_view` would give me a lot more information. Plus it helps in some unknown future state if we need to find a readable CV for example